Roys/app/src/main/java/com/dwg_karrier/roys/ListViewAdapter.java

```diff
@@ -0,0 +1,53 @@
+package com.dwg_karrier.roys;
+
+import android.content.Context;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.BaseAdapter;
+import android.widget.TextView;
+
+import java.util.ArrayList;
+
+/**
+ * Created by userpc on 2017-01-16.
```

Comment:
nit: userpc
Guess there must be some configuration you can put your name automatically.